packages/dashboard/src/components/grid/index.css

```diff
@@ -1,23 +1,26 @@
+@import "../../styles/variables.css";
+
 .container {
   position: absolute;
   height: 100%;
 
-  /* margin: 0.5em; */
-
   /**  establish a depth context */
   z-index: 1;
+  transition: all 50ms;
```

Comment:
IMO we should never use transition all because that introduces the possibility of accidental unwanted transitions later. Can we target the propertie(s) we want to transition?

Reply:
Ya good idea. I'll change these.